### PR TITLE
Create lab-10-2-mnist_nn_mlp.py : Added MLP for MNIST

### PR DESCRIPTION
**Pull request by jshin49**
When I was studying MLP with MNIST, I thought organizing the code this way was more easily understandable.

- I separated the model into a function called multilayer_perceptron_model to make it more generic.
- I also thought that organizing the weights and biases for each layer in a single place (dictionary) could make the code more generic and readable.
- I tried to follow KISS, but I thought that instead of using X and Y for variable names, sometimes names like 'images' and 'labels' could be more readable for people from non-mathematical backgrounds.

I would like your opinion on this PR.

### CODE REVIEW

**Comment by kkweon** [comment]
Looks good. 

I think we need a separate lecture to introduce several design patterns in Tensorflow.

**Comment by hunkim** [comment]
I would hold on this. Since it introduces a completely different pattern, I am not sure we need to introduce several patterns here.

I would just show one TF pattern, and go for tf.layers and Keras.
